src/lib/viewers/controls/annotations/annotations-button-region-tooltips/AnnotationsButtonRegionTooltips.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useContext, PropsWithChildren } from 'react';
+
+import { DependenciesContext } from '../../dependencies';
+
+import AnnotationsFlowTooltip from './AnnotationsFlowTooltip';
+import PersistentOnboardingBoxEditAnnotationsTooltip from './PersistentOnboardingBoxEditAnnotationsTooltip';
+
+export type Props = PropsWithChildren<{
+    isEnabled?: boolean;
+}>;
+
+export default function AnnotationsButtonRegionTooltips({ children, isEnabled }: Props): JSX.Element {
+    const { experiences } = useContext(DependenciesContext);
+    const { persistentOnboardingBoxEditAnnotations: persistentOnboardingBoxEditAnnotationsExperience } = experiences;
+
+    if (persistentOnboardingBoxEditAnnotationsExperience && persistentOnboardingBoxEditAnnotationsExperience.canShow) {
```

Comment:
I think we could use optional chaining here: 

```suggestion
    if (persistentOnboardingBoxEditAnnotationsExperience?.canShow) {
```

Reply:
I actually tried using optional chaining, but got Babel compilation error that this feature is off in this project and would need to be turned on. Very useful feature though, so it might be worthwhile considering turning it on.

I'm resolving this.